Description: Fix man page errors found by Lintian.
Author: Benjamin Drung <bdrung@debian.org>
Applied-Upstream: 0.9

--- a/src/clients/nycli/xmms2.1
+++ b/src/clients/nycli/xmms2.1
@@ -61,7 +61,6 @@
 .PP
 \fIposition\fR is a pattern to match playlist positions. (See \fBPOSITION PATTERN\fR below.)
 .SS General Commands
-.T
 \fBadd\fR [ \fB\-f\fR [\fB\-N\fR] [\fB\-P\fR] [\fB\-A\fR \fIkey=value\fR]... ] [\fB\-p\fR \fIplaylist\fR] [\fB\-n\fR | \fB\-a\fR \fIposition\fR] [\fB-o\fR \fIprop\fR[,...]] \fIarguments\fR...
 .br
 \fBadd\fR [\fB\-p\fR \fIplaylist\fR] [\fB\-n\fR | \fB\-a\fR \fIposition\fR] [\fB-o\fR \fIprop\fR[,...]] \fIpattern\fR...
@@ -152,7 +151,7 @@
 \fBhelp\fR [\fB\-a\fR] \fIcommand\fR
 .PP
 .RS 4
-Show the help on \fIcommand\fR. This is equivalent to \fBcommand -h\fR.
+Show the help on \fIcommand\fR. This is equivalent to \fBcommand \-h\fR.
 If \fIcommand\fR is an alias, show the alias definition.
 If no command is provided, list all available commands.
 .RE
@@ -777,7 +776,6 @@
 Match the complement of the pattern.
 .RE
 
-.T
 <pattern> <pattern>
 .br
 <pattern> \fBAND\fR <pattern>
--- a/src/clients/launcher/xmms2-launcher.1
+++ b/src/clients/launcher/xmms2-launcher.1
@@ -40,7 +40,7 @@
 .RB [\| -v \|]
 .RB [\| -- \|]
 .RI [\| XMMS2D_OPTION \|]
-.R \|...
+\|...
 .br
 .B xmms2-launcher -h
 .SH DESCRIPTION
